Gaijinizer patch/Map296.txt

```diff
@@ -29,81 +29,78 @@ are in a "breeding season".#GT
  よほど急いで獲物を捕まえようとしない限りは
 物を壊したり、ガラスをぶち破ったりもしなくなる。
 > CONTEXT: Map296/events/55/pages/0/25/Dialogue/other1 
-Because of the estrus, the ferocity
-also becomes less noticeable
-Unless you hurry to catch your prey
-I won't break things or break glass.#GT
+Because they're in heat, their brutal nature
+fades away, and when they're not chasing prey,
+they won't destroy buildings and break glass.#Amaboye
 > END STRING
 
 > BEGIN STRING
 まぁ、『物を壊さなくなる理由』は
 
 他にもデリケートなものがあるんだが……
 > CONTEXT: Map296/events/55/pages/0/30/Dialogue/other1 
-Well, "Why not stop breaking things" is
-
-There are other delicate things as well ...#GT
+Well, their "reasons to stop destruction" takes
+other, more delicate forms as well...#Amaboye
 > END STRING
 
 > BEGIN STRING
 見ろ………
 
 あれが、繁殖期のハーピーモドキだ。
 > CONTEXT: Map296/events/55/pages/0/58/Dialogue/other1 
-Look………
+Look over there...
 
-That is Beetle Harpy of the breeding season.#GT
+That's a Harpy Spawn in heat.#Amaboye
 > END STRING
 
 > BEGIN STRING
 胸を凝視する
 > CONTEXT: Map296/events/55/pages/0/62/Choice/0 
-Stare at the chest#GT
+Stare at her breasts#Amaboye
 > END STRING
 
 > BEGIN STRING
 毛並みを観察する
 > CONTEXT: Map296/events/55/pages/0/62/Choice/1 
-Observe the coat#GT
+Study her coat#Iffy/Amaboye
 > END STRING
 
 > BEGIN STRING
 あれ……？
 
 ハーピーモドキって、あんなに胸が大きかったっけ…？
 > CONTEXT: Map296/events/55/pages/0/65/Dialogue/kazuya 
-That......? Beetle Harpy, was the chest so big?#GT
+Huh? Wait, they weren't always that big, right?#Amaboye
 > END STRING
 
 > BEGIN STRING
 お前…………
 
 真っ先に見る場所そこかよ………
 > CONTEXT: Map296/events/55/pages/0/69/Dialogue/other2 
-My love............ 
-
-The first place to see...#GT
+My guy...
+You really went straight to looking there?#Amaboye
 > END STRING
```

Comment:
As much as it makes me laugh, "my guy" seems a little _too_ informal, lol. Need to think of a better way to phrase it...

Also, add two line breaks between the first and last line.